### DISCUSSION TITLE
Manual batch export to obsidian or omnivore

### DISCUSSION THREAD

**Discussion by xBlady** [usememos/memos#3993] [Ideas]
Hello!

I'm using memos as a brain dump thing, and i'd like to organize some of memos within obsidian later on when I catched something nice. It would be good to have option to export from memos  md files and do it in a way that wouldn't export things that i've previously exported and also allow for full export. 

Just an idea ;)

**Comment by RoccoSmit** [comment]
Exporting us to be a feature ([see](https://github.com/orgs/usememos/discussions/2805)), and was removed again ([see](https://github.com/usememos/memos/commit/a8ea28066e4296d2e379887296660ad85cc3b205)). Some context for the removal of the export functionality can be found [here](https://github.com/usememos/memos/issues/3878#issuecomment-2329013051)

Have a look at the Memos community [page](https://www.usememos.com/docs/contribution/community), there are 2 3rd party apps listed for Obsidian integrations

Alternatively, to create your own export functionality you can use either the API or webhooks to send changes to your memos to an external app.
